### ISSUE TITLE
karger.com

### ISSUE THREAD

**Issue by maximilianh** [ziplokk1/incapsula-cracker#2]
Hi, this doesn't seem to work at all with karger.com... any ideas?

**Comment by maximilianh** [comment]
example link: http://www.karger.com/Article/FullText/437330

**Comment by ziplokk1** [comment]
Hello Maximilian,

Incapsula seems to have updated within the last couple weeks and I've been
working on getting the module updated as well. It seems like your example
link for karger.com is using the new incapsula resource.

I am working on the fix in the 'hotfix' branch though unfortunately its
still not working with the link you provided.

I will look into the issue and try to fix it as soon as possible.

Thank you,
Mark

**Comment by ziplokk1** [comment]
I've looked into the issue and it seems like karger.com is a special case. The unblocking script works, but doesn't detect that it has; since even though it's bypassed the robot check, there's still the `<meta name="robots">` tag. This tag is what the unblocker uses to detect whether the resource is blocked in the first place.

For your case, I'll have to look around and see if there's a way that I can identify whether the resource is blocked or not without depending on that "meta" tag. Another issue that I'm having with your URL is that it seems like it requires a login to view the FullText since it keeps redirecting me to http://www.karger.com/Article/Abstract/437330.

If you can make do with the script thinking that the resource is still blocked, I'd recommend checking out the hotfix branch for now until I can get everything in that branch working and I'm able to merge it into master.

**Comment by maximilianh** [comment]
Wow, great! Yes, nevermind the redirect, if you get to the redirect, that's
already great, that means you're past the robot check.
many Thanks!

**Comment by ziplokk1** [comment]
No problem, glad I could help.
- Mark
